src/SendingPromise.cpp

```diff
@@ -50,16 +50,16 @@ uint16_t SendingPromise::send(const String& value) {
   strcat(topic, Interface::get().getConfig().get().deviceId);
   strcat_P(topic, PSTR("/"));
   strcat(topic, _node->getId());
-  strcat_P(topic, PSTR("/"));
-  strcat(topic, _property->c_str());
-
   if (_range.isRange) {
     char rangeStr[5 + 1];  // max 65536
     itoa(_range.index, rangeStr, 10);
     strcat_P(topic, PSTR("_"));
     strcat(topic, rangeStr);
   }
```

Comment:
Ranges are not part of Homie 3?

Reply:
Ranges are how Node arrays are called in homie-esp8266, it's just an interval attribute name.

Reply:
@euphi 
How can we do that ?

Reply:
Just add after line 57 (`strcat(topic, rangeStr);`):

```C++
  _range.isRange = false;
  _range.index = 0;
```

It is a quick workaround. I think the range-mechanism has to be rewritten.

Reply:
@euphi 
Thanks